app/main.py

```diff
@@ -1,6 +1,21 @@
+import os
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    key = os.getenv("API_KEY")
+    response = requests.get(
+        URL,
+        params={"q": CITY, "key": key},
+    )
+    print(
+        f"Actual temperature in Paris: {response.json()['current']['temp_c']}"
+    )
```

Comment:
The 'get_weather' function has a side effect: it prints the weather. It would be better to separate the concerns: one function to get the weather and another one to print it. This way, the 'get_weather' function could be reused in other parts of the code without printing the weather.